rigpa.el

```diff
@@ -64,17 +64,12 @@
 (require 'rigpa-history-mode)
 (require 'rigpa-tower-mode)
 
-;; define face for use in meta modes
+;; define face for use in meta modes (make-face 'rigpa-face)
 (make-face 'rigpa-face)
 (set-face-font 'rigpa-face
-               (cond ((find-font (font-spec :name "Consolas"))
-                      "Consolas")
-                     ((find-font (font-spec :name "Courier New"))
-                      "Courier New")
-                     (t (let ((font (face-attribute 'default :font)))
-                          (if (eq font 'unspecified)
-                              'unspecified
-                            (font-get font :name))))))
+               (or (find-font (font-spec :name "Consolas"))
+                   (find-font (font-spec :name "Courier New"))
+                   (face-attribute 'default :font)))
```

Comment:
I see, so `set-face-font` accepts `unspecified` even though [`font-get` does not](https://github.com/countvajhula/rigpa/pull/7/files)?

Reply:
That seems to be the case from my experimentation, although the docstrings don't make that super clear. 

From `font-get`:
```
FONT is a font-spec, a font-entity, or a font-object.
```

From `set-face-font`:
```
FONT can be a string, a font spec, a font entity, a font object,
or a fontset.
```

But it looks like `set-face-font` is ultimately forwarding its FONT argument to `set-face-attribute`, which does say it can take `unspecified`:
```
As an exception, to reset the value
of some attribute to unspecified in a way that overrides the
non-unspecified value...
```